### DISCUSSION TITLE
Vorschlag für neues Wort Star Citizen Deutsch

### DISCUSSION THREAD

**Discussion by BTSVDanielk** [rjcncpt/StarCitizen-Deutsch-INI#535] [Übersetzungsvorschläge]
Wenn ich als Star Citizen Spieler im Raumschiff bin und die Taste I für Inventar drücke, dann habe ich die Möglichkeit alle Items, die in mein Raumschiff gelagert sind, in meinen Rucksack / Home Inventar zu verschieben oder umgekehrt. Voraussetzung ist natürlich das man im Raumschiff im Hangar oder an einen Outpost ist. In eurer Übersetzung steht auf der Taste zum Verschieben der Items "Alles". Ich würde diesen Button mit in den Namen "verschieben“ oder "Alles verschieben“ umbenennen. Im Englischen steht an dieser Stelle glaube ich "move".

**Comment by BTSVDanielk** [comment]
![Star Citizen deutsch](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/73cf2b45-c5b6-4003-b5f1-3c205b7e2f21)

**Comment by BTSVDanielk** [comment]
Beim Drücken der Taste wird alles in das gegenüberliegende Inventar verschoben, daher bin ich für das Wort "Alles verschieben" sofern es von der größe des Textes in das Button Feld passt.

**Comment by MaxM1211** [comment]
Hey @BTSVDanielk 

Danke für den Vorschlag.
Wie du es schon geschrieben hast, stimmt das mit dem `Move All`. 
Im Moment steht in unserer Übersetzung `Alles versch.`, so wie es aussieht wird aber nur `Alles` angezeigt.

Ich experimentiere mal ein bisschen herum, was alles so in den Button passt.
Falls du "kurze" Vorschläge hast, kannst die gerne gleich hier posten.

Beste Grüße

**Reply by MaxM1211** [reply]
Tatsächlich waren wir mit `Alles versch.` ein Zeichen zu lang. Ich würde Vorschlagen den Text zu `Alle versch.` zu ändern.

**Reply by BTSVDanielk** [reply]
Das dachte ich mir schon, dass mein vorschlag zu lang ist.  Alle versch. hört sich gut an.
Danke nochmal an das gesamte Team für die Übersetzung.

**Reply by MaxM1211** [reply]
Perfekt, das kommt dann sobald der PTU auf den LIVE kommt.

Danke dir für die Rückmeldung.
